themes/custom/essex/templates/content/node--localgov-news-article--full-anonymous.html.twig

```diff
@@ -112,9 +112,11 @@
 
           {{ content|without('localgov_news_date', 'localgov_news_categories', 'localgov_news_related', 'sign_in') }}
 
-          {% if node.localgov_news_related.value %}
-            <h2 class="news-article__related-title">{{ 'Related news'|t }}</h2>
-            {{ content.localgov_news_related }}
+          {% if view_mode is not same as('full_anonymous') %}
```

Comment:
This condition shouldn't be necessary. This template is only for the full_anonymous display mode.

Reply:
I think I've already removed these lines on release/202311 branch.